### PR TITLE
Crashlytics ios

### PR DESCRIPTION
**Pull request by Javasamurai**

### CODE REVIEW

File: build/index.js [hunk 1]
```diff
@@ -18,6 +19,18 @@ exports.onCreateProject = function (api, app, config, cb) {
...
-}; 
+  if (config.target === 'native-android') {
+    return copyCustomRules(outputPath)
+      .then(cb);
+  } else if (config.target === 'native-ios') {
+    var buildScipt = path.join(__dirname, '../ios', 'buildScript'),
+      manifest = app.manifest.ios,
+      scriptContent = fs.readFileSync(buildScipt, 'utf-8');
+
+    outputPath = path.join(outputPath, 'xcodeproject');
```
Comment (bijoshtj) [inline build/index.js]: i think this path can be obtained from some property in app/config

File: build/index.js [hunk 2]
```diff
@@ -18,6 +19,18 @@ exports.onCreateProject = function (api, app, config, cb) {
...
+    return fs.outputFileAsync(path.join(outputPath, 'resources', 'extra'),
+      scriptContent.toString() + ' ' +manifest.crashlyticsKey +
+        ' ' + manifest.crashlyticsExtraKey, 'utf-8')
```
Comment (bijoshtj) [inline build/index.js]: what will be crashlyticsExtraKey ?

Comment (Javasamurai) [reply build/index.js]: crashlyticsExtraKey is extra key assigned to run the script with the crashlytics app key.

**Review by rampr — APPROVED** [review]

File: build/index.js (hunk 1 above)
Comment (Javasamurai) [reply build/index.js]: This is xcode project path which is fixed for devkit.It's been used in native-ios too.

**Review by bijoshtj — APPROVED** [review]

**Pull request merged**